### ISSUE TITLE
any chance that logos will be updated sometime ?

### ISSUE THREAD

**Issue by helmarw** [authenticatorplus/logos#234]
thnx
Helmar

**Comment by mKoonrad** [comment]
I think dev abandoned project... 🙄 cc @MufriA

**Comment by MufriA** [comment]
busy with other projects, tried to hire for this work but with one time fee it's not scaling well, I will do when I get some free time, till then you can add icon locally it will show it for you

**Comment by helmarw** [comment]
how can i add my own icons? when i go to edit icon i can only select from the list or send a request, there is no option where i can select my own file
thnx
Helmar

**Comment by mKoonrad** [comment]
@MufriA Maybe I can help? By the way authplus.db backup file contain also own uploaded icons?

**Comment by mKoonrad** [comment]
@helmarw When you go to edit icon first square at the top on left - custom logo

**Comment by helmarw** [comment]
on my screen its Google

<img width="320" alt="Bildschirmfoto 2019-11-07 um 22 29 24" src="https://user-images.githubusercontent.com/10408121/68429383-41513600-01ae-11ea-89cc-894bb103a6eb.png">

**Comment by mKoonrad** [comment]
@helmarw Sorry, I was talking about Android version.

**Comment by helmarw** [comment]
oki, thought so, no problem ;)

**Comment by MufriA** [comment]
Same steps in Android too, long press any account, tap icon image, tap custom logo, which is left most top

**Comment by MufriA** [comment]
> @MufriA Maybe I can help? By the way authplus.db backup file contain also own uploaded icons?

no it won't be synced it's local only, adding db will make db heavy and create sync issues.

Thanks for extending help, how do you think you can help, since this is not open source project happy to explore options

**Comment by helmarw** [comment]
> Same steps in Android too, long press any account, tap icon image, tap custom logo, which is left most top

i don't know what you mean. there is no option for selecting custom logo in iOS, see my screenshot I uploaded before, also long press doesnt change anything

**Comment by mKoonrad** [comment]
@MufriA Without sharing with me credentials I can only prepare to you any files pack in specific format. Btw issues in GitHub are very easy to handle this process. But you still need a little time to update it.

Btw you should accept only images in PNG format with transparent background. They're looks better.

**Comment by richboos** [comment]
We really need to ability to add custom logos. Allegedly the android version has this, but iOS does not? I'm having trouble wondering why it would be designed in a way that every future website has to pass through a single person on planet Earth. Seems like a bottleneck. Why not keep it open-ended and let iOS users supply their own images?

**Comment by mKoonrad** [comment]
I'd like also remind that Authenticator Plus is paid app. It should be supported. Now users don't have logos support, last Android update was 31.12.2018, iOS 18.03.2018...

**Comment by ghost** [comment]
@MufriA Hello. 
Your application has no competitors. It’s a pity that you don’t have time for it. But could you at least occasionally update it? Some bugs already appear on new versions of the android, the application crashes several times when entering personalization. It is necessary to correct the localization and bring out the text for translation. Clean from Google+. There is not enough other way to synchronize, for example by sftp.

**Comment by zQueal** [comment]
@MyTimeKill These all seem to be local issues. I've not experienced this issue on any one of my devices which run Authenticator and Android.

@MufriA If you could commit to pulling the icons via a repository via naming conventions or something, then you could easily outsource the icon issue to the community. No need to pay someone to do it. This is the only Authentication app that I use, but icon management is a huge hole in usability.

There are a great deal of excellent icon sets for popular services.

 * https://www.iconfinder.com/iconsets/logos-brands-3
 * https://www.iconfinder.com/iconsets/social-media-2169

These two alone cover a great deal of 2FA compliant apps--all completely free under creative commons.

**Comment by JadeTusk** [comment]
> 
> 
> @MufriA Hello.
> Your application has no competitors. 

Not true, try [Authy](https://play.google.com/store/apps/details?id=com.authy.authy&hl=en) or [Authenticator Pro](https://play.google.com/store/apps/details?id=me.jmh.authenticatorpro&hl=en_US&gl=US). 
This app has been clearly abandoned, after he took the money of course.
